libs/tailwindcss-config/src/theme.js

```diff
@@ -16,69 +22,67 @@ module.exports = {
       yellow: '#EDFF22',
       pink: '#FF2D5E',
       green: '#00F780',
+      red: '#FF261A',
     },
     red: {
       DEFAULT: '#ED1515',
-      transparent: 'rgba(255, 38, 65, 0.3)',
-      vega: '#FF261A',
       dark: '#EB001B',
+      bar: 'rgba(47, 246, 139, 0.45)', // #2FF68B 45%
     },
     green: {
       DEFAULT: '#26FF8A',
-      transparent: 'rgba(38, 255, 138, 0.3)',
       dark: '#008545',
-      vega: '#00F780',
+      bar: 'rgba(47, 246, 139, 0.45)', // #2FF68B 45%
     },
     text: '#C7C7C7',
     deemphasise: '#8A9BA8',
     white: {
       DEFAULT: '#FFF',
-      '02': 'rgba(255, 255, 255, 0.02)',
-      '05': 'rgba(255, 255, 255, 0.05)',
-      10: 'rgba(255, 255, 255, 0.10)',
-      25: 'rgba(255, 255, 255, 0.25)',
-      40: 'rgba(255, 255, 255, 0.40)',
-      60: 'rgba(255, 255, 255, 0.60)',
-      80: 'rgba(255, 255, 255, 0.80)',
-      95: 'rgba(255, 255, 255, 0.95)',
-      100: 'rgba(255, 255, 255, 1.00)',
+      strong: '#FFF',
```

Comment:
Chore (blocking): We've begun to rely on opacity with these white and black colours. I think some solid greys are a great idea, but we should keep semi-opaque colours too

Reply:
You can get semi opaque by `className="text-black text-opacity-20"`
Supported opacity levels will be taken from theme setup which already has it fine grained
```
 opacity: {
    0: '0',
    2: '0.02',
    5: '0.05',
    10: '0.1',
    15: '0.15',
    20: '0.2',
    25: '0.25',
    30: '0.3',
    35: '0.35',
    40: '0.4',
    45: '0.45',
    50: '0.5',
    55: '0.55',
    60: '0.6',
    65: '0.65',
    70: '0.7',
    75: '0.75',
    80: '0.8',
    85: '0.85',
    90: '0.9',
    95: '0.95',
    100: '1',
  },
```